### PR TITLE
Italian Translation

### PR DESCRIPTION
**Pull request by Davis0308**
Italian Translation

### CODE REVIEW

**Comment by Henryws** [comment]
Follow all the [instructions](https://github.com/pacstall/website/blob/master/client/src/locale/HOWTO.md)

**Comment by Davis0308** [comment]
> Follow all the [instructions](https://github.com/pacstall/website/blob/master/client/src/locale/HOWTO.md)

I have updated `locale.ts`

File: client/src/locale/locale.ts [hunk 1]
```diff
@@ -149,6 +153,7 @@ export const localeFlags: Record<keyof typeof translations, string> = {
     'es-ES': 'ES 🇪🇸',
     'pt-BR': 'PT 🇧🇷',
     'sv-SE': 'SV 🇸🇪',
+    'it-IT': 'IT 🇮🇹',
```
Comment (Henryws) [inline client/src/locale/locale.ts:156]:
```suggestion
'it-IT': 'IT 🇮🇹',
```

**Review by Henryws — CHANGES_REQUESTED** [review]

**Comment by Henryws** [comment]
Alright I'll merge later tonight

**Pull request merged**